src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/MediationSuccessfulRespondentNotificationHandler.java

```diff
@@ -43,17 +54,91 @@ public String camundaActivityId(CallbackParams callbackParams) {
 
     private CallbackResponse notifyRespondent(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
-        if (caseData.isRespondent1NotRepresented() && caseData.getRespondent1().getPartyEmail() != null) {
-            notificationService.sendMail(
-                caseData.getRespondent1().getPartyEmail(),
-                addTemplate(caseData),
-                addProperties(caseData),
-                String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference()));
+        Boolean isCarmEnabled = featureToggleService.isCarmEnabledForCase(caseData.getSubmittedDate());
+        if (isCarmEnabled) {
+            String claimId = caseData.getCcdCaseReference().toString();
+            String referenceTemplate = String.format(REFERENCE_TEMPLATE, claimId);
+            String defendantEmail = caseData.getRespondent1().getPartyEmail();
```

Comment:
this will work for LiP defendant but if it's LR defendant it should be caseData.getRespondentSolicitor1EmailAddress()